docs/src/main/asciidoc/hibernate-orm-panache.adoc

```diff
@@ -1,326 +1,752 @@
 ////
-This guide is maintained in the main Quarkus repository
+This guide is maintained in the main {ProductName} repository
 and pull requests should be submitted there:
 https://github.com/quarkusio/quarkus/tree/master/docs/src/main/asciidoc
 ////
-= Quarkus - Simplified Hibernate ORM with Panache
+= Using simplified Hibernate ORM with Panache
+
 
 include::./attributes.adoc[]
 :config-file: application.properties
+//:COMM:
+:ENT:
 
-Hibernate ORM is the de facto JPA implementation and offers you the full breadth of an Object Relational Mapper.
-It makes complex mappings possible, but it does not make simple and common mappings trivial.
-Hibernate ORM with Panache focuses on making your entities trivial and fun to write in Quarkus.
 
-== First: an example
+As a Java developer, you can use Hibernate ORM with Panache to simplify persistence with your {ProductName} applications.
```

Comment:
I feel like `As a Java developer` is a bit redundant because Quarkus ATM is aimed at Java devs, so we can assume that's the sort of person we're talking about.

Reply:
This is the form we use with product stories so that the user knows the target audience.

Reply:
So perhaps we should make this conditional on product? Project users will know Quarkus is for Java devs.

Reply:
OK